### ISSUE TITLE
Can you share the testing videos in the paper for quantitative results?

### ISSUE THREAD

**Issue by Wuziyi616** [Red-Fairy/argus-code#3]
Can you share the 101 360-deg videos and their corresponding camera trajectories mentioned in Sec.4.1? If I understand correctly, you crop perspective videos from them as the input to your 512x1024 model. What's the perspective video's resolution & how long are they (how many frames / seconds)?

A few more questions regarding metrics:
1. You mentioned that FVD is computed on the 360-deg panorama video. However, I believe FVD only accepts square videos as input (224x224 or smth similar). Do you just resize it to this resolution and apply FVD?
2. VBench is computed on 4 projected square perspective videos. What's the resolution / FOV of the perspective videos here? Given that your pano video is 512x1024, is the perspective video obtained with FOV=90 and thus the resolution 256x256?
3. I'm a bit confused by the description that "PSNR and LPIPS are computed within masked regions of visible directions and aggregated over frames." Did you 1) for each frames, get the perspective-on-panorama mask based on camera pose and FOV (basically the visible regions), then 2) compute LPIPS / PSNR on that region? If this is the case, then this metric sounds like measuring input view reconstruction?
- OR, did you compute the perspective-on-panorama mask on ALL frames, take a **union** on ALL frames, and compute LPIPS / PSNR on the aggregated visible regions? But then the aggregated mask will be irregular (non-square), how to compute LPIPS non-square regions? Do you just mask out invisible pixels (setting them to 0?) and then compute PSNR?

Thank you so much for your time!

**Comment by Red-Fairy** [comment]
I added the google drive link to the 101 evaluation clips and their trajectories. The resolution is 512x1024 and they're sampled from a set of youtube videos at 5fps and last 5 second (25 frames in total). You can use the --cached_motion_path argument to read these files. Please refer to the code [here](https://github.com/Red-Fairy/argus-code/blob/77f832e88189f10190b0b428840bb6af92ae4362/dataset/video_dataset.py#L202C1-L210C54).

For the questions:
1. Yes. I just do simple resize
2. Your understanding is correct.
3. Yes. But since the camera viewing direction might be changing, it's not necessarily reconstruction (some region are unseen)

**Comment by Wuziyi616** [comment]
That's very cool. Thanks for the reply. I wonder if you have the original ID of the 101 testing videos though, cause I'm fine-tuning a base video model whose FPS is >> 5FPS.

For (3), just wanna double-check, you took the **union** of per-frame masks to get the visible regions? To compute masked LPIPS / PSNR, you just set non-visible regions to 0 (the value range here is [0, 1] right?)

**Comment by Red-Fairy** [comment]
Yes. I took the union

**Comment by Wuziyi616** [comment]
Thanks for confirming it (at such a late night!). Though I guess you missed the first part of the question:
> I wonder if you have the original ID of the 101 testing videos though, cause I'm fine-tuning a base video model whose FPS is >> 5FPS.

**Comment by Red-Fairy** [comment]
Sorry, but I cannot find the exact original video IDs. I recall I downloaded some additional videos from YouTube in the Travel & Events category outside the training set, separate into 5-sec clips, and manually selecting a few for use. What I have now is a set of 817 video ids that these videos are sourced from, but I cannot find the exact video ids for use and their start/end time.

Under this circumstance, for evaluation purposes, I would recommend curating your own dataset to ensure a fair comparison if your model is higher than 5fps.

[video_ids.txt](https://github.com/user-attachments/files/22805619/video_ids.txt)

**Comment by Wuziyi616** [comment]
Got it. Thanks for looking into this!

**Comment by Wuziyi616** [comment]
> (evaluation clips) they're sampled from a set of youtube videos at 5fps and last 5 second (25 frames in total)

Hi Rundong, sorry for bothering again. I'm trying to retrieve the 101 videos from the list of video IDs you sent. Are you sure the clips are sampled from the last 5s in each video? Looking at some clips in the Google Drive link, they look pretty much the same person. I wonder if they are actually from the same video.

<img width="1192" height="616" alt="Image" src="https://github.com/user-attachments/assets/943b2216-4a48-4f28-a14f-825046e53341" />

**Comment by Red-Fairy** [comment]
Oh thanks for pointing it out! I just noticed that I mixed a few videos captured by us (actually my advisor during his faculty campus visit lol). Also, videos are not necessarily from the last 5 second. They might be anywhere in the entire clip. Really sorry I didn't save the exact position of these clips in the videos

Those videos captured by ourselves are [here](https://drive.google.com/drive/folders/1Iqr4gkn8ZCreThP_xvzCZ0mGOG91XZiI?usp=drive_link)

**Comment by Wuziyi616** [comment]
Lol no worries at all! So I guess I'll just use a random subset from the "Travel & Events category" videos you sent. I'll pick those with human activities as the low-FPS clips you sent mostly belong to this category. From my experience these metrics (PSNR, VBench) should be similar across a similar subset.

**Comment by Wuziyi616** [comment]
Hi Rundong sorry for re-openning this again but, there is just one more thing I want to double-check: the results you used for computing quantitative results (e.g. Tab.1), are they after VEnhancer post-processing? What about those qualitative results?

I'm re-running Argus on the 101 videos I randomly sampled for a fair comparison (proving it's not that I select simple videos). And I want to report a row in the Table as "Reproduced Argus on our test set" or smth like that.

**Comment by Wuziyi616** [comment]
Some videos I got look like this (without any post-processing), not sure if that looks reasonable to you.

https://github.com/user-attachments/assets/30760248-f913-483f-8756-5695abcb7d5e

https://github.com/user-attachments/assets/6e874a24-cab3-4eda-9f78-f3a080f8976c

https://github.com/user-attachments/assets/ed2734c9-8daa-438f-8504-8c9bdca9e08e

**Comment by Red-Fairy** [comment]
I think the reproduction is correct. For the results, the quantitative ones are before enhancement, but the qualitative ones are after enhancement

**Comment by Wuziyi616** [comment]
Thank you!
